src/commands/kv/bucket/mod.rs

```diff
@@ -36,6 +36,10 @@ pub fn directory_keys_values(
         let entry = entry.unwrap();
         let path = entry.path();
         if path.is_file() {
+            if print_files {
+                message::working(&format!("Processing {}...", path.display()));
```

Comment:
Should "Processing" be "Uploading"? We aren't doing any processing on each file.

Reply:
On second thought (personal opinion incoming): I think I'd like to just have each file name on its own line without `Processing` or the `...`.

Reply:
Heh Sven suggested switching to Processing instead of Parsing! I think either is okay... I have no strong opinions on which verb we use. I can get rid of the ellipses easy.

Reply:
Yeah - I feel like it might be redundant to have any verb (not hugely important) but seems like we could get away with each line just being the file name by itself with no frills (maybe even no emoji for easy copy pasting?)

Reply:
How about I use the word "preparing"? I guess we are preparing since we are base64 encoding all file contents :)